src/cleanfigure.m

```diff
@@ -156,32 +156,28 @@ function pruneOutsideBox(meta, handle)
         return;
     end
 
-    %hasLines = ~strcmp(lineStyle,'none') && lineWidth>0.0;
-    %hasMarkers = ~strcmp(marker,'none');
-    hasLines = true;
-    hasMarkers = true;
+    % Check if the plot has lines
+    hasLines    = ~strcmp(get(handle, 'lineStyle'),'none')...
```

Comment:
Please use `CamelCase` for the keywords `LineStyle` and `LineWidth` (as per the MATLAB convention).